### PR TITLE
Basic superfund servlet

### PR DESCRIPTION
**Pull request by gusteran**
Set up a servlet to respond to GET requests
Pulls data from EPA API
Includes unit tests

run tests with 'mvn test'

### CODE REVIEW

**Comment by gusteran** [comment]
@googlebot I signed it!

**Comment by gusteran** [comment]
I really hope that whole rebase fixed it but I think it did not

**Comment by gusteran** [comment]
I had just forgotten to register my Github account and did not actually have to rebase but I learned something at least

File: src/main/java/com/google/sps/data/SuperfundSite.java [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        String status,
+        double lattitude,
+        double longitude){
+        this.name = name;
+        this.score = score;
+        this.state = state;
+        this.city = city;
+        this.county = county;
+        this.status = status;
+        this.lattitude = lattitude;
```
Comment (natalie-abreu) [inline src/main/java/com/google/sps/data/SuperfundSite.java:30]: Should be "latitude"

**Pull request merged**